style.css

```diff
@@ -0,0 +1,35 @@
+.wrapper { 
+position: relative;
+width: 100%;
+flex-direction: row;
+display: table;
+}
+
+span{
+display:table-cell;
```

Comment:
Don't use table stuff for layouts, it is very old techniques, and you may have problems with it.